File: examples/distribution/example_ks.py
import scipy.stats as stats, statsmodels.stats.diagnostic as ssd
Boston <- r["mmstat4::ghload"]("Boston200.rds")
data = boston_df['medv'].dropna()
# Kolmogorov-Smirnov & Lilliefors test 
ks = stats.kstest(data, 'norm', args=(np.mean(data), np.std(data)))
print(ks)
lf = ssd.lilliefors(data)
print(lf)

File: examples/distribution/example_ks.R
Boston <- mmstat4::ghload("Boston200.rds")
# Kolmogorov-Smirnov test
ks.test(Boston$medv, "pnorm", mean=mean(Boston$medv),
sd=sd(Boston$medv))
# Lilliefors test
library("nortest")
lillie.test(Boston$medv)
